tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -634,10 +634,23 @@ public void VerifyTestProgramFullManifest()
                     }
                   }
                 },
+                "mongodb-password": {
+                  "type": "parameter.v0",
+                  "value": "{mongodb-password.inputs.value}",
+                  "inputs": {
+                    "value": {
+                      "type": "string"
+                    }
+                  }
+                },
                 "mongodb": {
                   "type": "container.v0",
-                  "connectionString": "mongodb://{mongodb.bindings.tcp.host}:{mongodb.bindings.tcp.port}",
+                  "connectionString": "mongodb://admin:aspire123@{mongodb.bindings.tcp.host}:{mongodb.bindings.tcp.port}",
```

Comment:
This should be referencing the parameter ... we should never have a literal secret here.